code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -363,6 +363,36 @@
 	for(var/i in 1 to initial_amt)
 		new pill_type(src)
 
+/obj/item/storage/pill_bottle/njoy/red
+	name = "bottle of red Njoy pills"
```

Comment:
So is Njoy written uppercase or lowercase? It's uppercase here and lowercase there - decide and standartize it

Reply:
> Capitalize brand names that are commonly used to refer to both the name brand and generic products.

Njoy yes

> The names of chemicals are not capitalized unless they are the first word of a sentence.

Suppressital no

Reply:
All reagents are capitalized in code so they should be kept as-is because you only see them on UI (e.g. Urist McVagabond has 30 Tricordrazine in bloodstream)

Reply:
Adressed.